lib/whenever/cron.rb

```diff
@@ -147,7 +147,7 @@ def parse_as_string
           return (timing << i) * " " if string.downcase.index(day)
         end
 
-        raise ArgumentError, "Couldn't parse: #{@time}"
+        raise ArgumentError, "Couldn't parse: \"#{@time}\""
```

Comment:
Why not `"Couldn't parse: #{@time.inspect}"`, which doesn't assume this is a string?